app/src/ingestion/pdf_stylings.py

```diff
@@ -319,15 +319,11 @@ def _create_phrase(self, parent_node: Element | None, child: Text) -> Phrase | N
         return Phrase(text=child.data, bold=bolded)
 
 
-class BemTagExtractor(TagExtractor):
+class GenericTagExtractor(TagExtractor):
```

Comment:
I wouldn't call this Generic since it was implemented specifically for BEM pdfs. Either remove it entirely or leave it as is.

Reply:
Removed this file `pdf_stylings.py` as it was only used by BEM